unreleased/s3mapprovider-new-component.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
```

Comment:
Since this is a new component it will [require a sponsor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-new-components) when an upstream PR is created.  I can be that sponsor.  We've skipped some of that process in writing code before creating an issue proposing the new component, but we're probably fine here as this is not the type of component that process is typically aimed at.

Reply:
Gotcha, thanks you a lot :)